app/main.py

```diff
@@ -1 +1,8 @@
-# write your code here
+def copy_file(file_names: str) -> None:
+    command, original, copy = file_names.split()
+    if original == copy:
+        exit(0)
+    if command == "cp":
+        with open(original, "r") as original, open(copy, "w") as copy:
+            lines = original.readlines()
+            copy.writelines(lines)
```

Comment:
you can try to remove the redundant variable and make the same without it in one line